src/macros.rs

```diff
@@ -610,7 +609,7 @@ macro_rules! service {
         impl<S> SyncServiceExt for S where S: SyncService {}
 
         #[allow(unused)]
-        #[derive(Debug)]
+        #[derive(Clone, Debug)]
         /// The client stub that makes RPC calls to the server. Exposes a blocking interface.
         pub struct SyncClient(FutureClient);
```

Comment:
Instead of just wrapping `FutureClient` directly, let's wrap it in a mutex. Then sync callers can still `SyncClient` across threads to make concurrent requests.